tests/functional/prometheous/test_prometheus_validations_file.py

```diff
@@ -333,3 +330,70 @@ def test_prometheus_pv_resize(self):
             pod_name=pod_name, pvc_name=pvc_name,
             filename="secondfilename", dirname="seconddirname",
             metric_data=resize_metrics, operation="create")
+
+    @pytest.mark.tier2
+    def test_prometheus_volume_metrics_on_node_reboot(self):
+        """Validate volume metrics using prometheus before and after node
+        reboot"""
+
+        # Create PVC and wait for it to be in 'Bound' state
+        pvc_name = self.create_and_wait_for_pvc()
+        pod_name = openshift_ops.oc_create_tiny_pod_with_volume(
+            self._master, pvc_name, "autotest-volume",
+            image=self.io_container_image_cirros)
+        self.addCleanup(openshift_ops.oc_delete, self._master, 'pod', pod_name,
+                        raise_on_absence=False)
+        openshift_ops.wait_for_pod_be_ready(
+            self._master, pod_name, timeout=60, wait_step=2)
+
+        # Write data on the volume and wait for 2 mins and sleep is must for
+        # prometheus to get the exact values of the metrics
+        self.cmd_run("oc rsh {} touch /mnt/file{{1..1000}}".format(pod_name))
+        time.sleep(120)
+
+        # Fetching the metrics and storing in initial_metrics as dictionary
+        initial_metrics = self._get_and_manipulate_metric_data(
+            self.metrics, pvc_name)
+        openshift_ops.switch_oc_project(
+            self._master, self.storage_project_name)
+
+        # Get the hostname to reboot where the pod is running
+        pod_info = openshift_ops.oc_get_pods(self._master, name=pod_name)
+        node_for_reboot = pod_info[pod_name]['node']
+
+        # Perform node reboot and wait for ssh connection
+        node_ops.node_reboot_by_command(
+            node_for_reboot, timeout=600, wait_step=10)
```

Comment:
why this increased timeout delay?

Reply:
The default timeout also 600 seconds from the library. I just mentioned it.

Reply:
this is default value from library, so no need to mention it again in tc.

Reply:
ack will remove it.